oap-server/server-alarm-plugin/src/main/java/org/apache/skywalking/oap/server/core/alarm/provider/RulesReader.java

```diff
@@ -203,4 +206,25 @@ private void readCompositeRuleConfig(Rules rules) {
             }
         });
     }
+
+    /**
+     * Read composite rule config into {@link DingtalkSettings}
```

Comment:
Be careful when copy-and-pasting
```suggestion
     * Read alarm rule config into {@link DingtalkSettings}
```

Reply:
done